data/dev.jonmagon.kdiskmark.helper.policy

```diff
@@ -13,6 +13,7 @@
       <message xml:lang="fr">Autorisation est requise pour mesurer la performance des disques</message>
       <message xml:lang="hu">Hitelesítés szükséges a lemezek teljesítményértékeléséhez</message>
       <message xml:lang="ja">特権的な KDiskMark の実行ヘルパー</message>
+      <message xml:lang="de">Authenticatie vereist om schijven te kunnen benchmarken</message>
```

Comment:
de is for Deutsch

Reply:
Sorry, my bad. I copied over the entire string to make sure the formatting was kept, but I accidentally forgot to replace de with nl.